### DISCUSSION TITLE
Simulator access and challenge participation

### DISCUSSION THREAD

**Discussion by MatthewHowe** [learn-to-race/l2r#3] [Q&A]
Hi! 
The simulator looks great, I’m very excited to get to use it. I have a few questions:
-How long does it take to get access to the simulator?
-Are there any updates on the challenge, participation requirements, how to sign up?
-Is there going to be some sort of launch event for the challenge?
Cheers!

**Comment by jonfranc** [comment]
Following up for others that land here: the inaugural [Learn-to-Race Autonomous Racing Virtual Challenge](https://www.aicrowd.com/challenges/learn-to-race-autonomous-racing-virtual-challenge) has been officially launched and is accepting participation!
